plexapi/base.py

```diff
@@ -253,7 +357,9 @@ def fetchItems(self, ekey, cls=None, container_start=None, container_size=None,
             headers['X-Plex-Container-Size'] = str(container_size)
 
             data = self._server.query(ekey, headers=headers)
-            subresults = self.findItems(data, cls, ekey, **kwargs)
+            if not data:
+                return []
```

Comment:
Is this necessary?

Reply:
`.query` can return `None` and `.findItems` does not accept `None`. so it should be handled in some way. returned empty list as `.fetchItems` returns a list

Reply:
on a second thought maybe returning `result` is a better choice instead of empty list? or should raise error? what is the preferred way to handle no date? other choice is to make query raise an error instead of returning `None`